fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/panel/panel.cfg

```diff
@@ -175,3 +175,11 @@ texture=NO_TEXTURE
 background_color=0,0,0
 
 htmlgauge04=A380X/ExtrasHost/extras-host.html
+
+[VCockpit22]
+size_mm=0,0
+pixel_size=0,0
+texture=NO_TEXTURE
+background_color=0,0,0
+htmlgauge00=A380X/ANC/anc.html, 0,0,1,1
+
```

Comment:
Adding this would be unnecessary if it is incorporated as part of ``extras-host`` instead

Reply:
Yes I will incoporate it in the extras-host